### DISCUSSION TITLE
realtime subscribe request hook

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#4153] [Q&A]
How can we called a hook   when a particular collection realtime subscribe request send from frontend?

```
onRealtimeAfterSubscribeRequest((e) => {
    const record = e.client.get("authRecord")
 record.set("active", true);
        $app.dao().saveRecord(record)
},"example")
```

This code not work

**Comment by ganigeorgiev** [comment]
I don't understand what you are asking and what the above is supposed to do.
If you are trying to inspect the client subscriptions, then you can try checking `e.subscriptions`.

**Comment by apujayed** [comment]
i supposed to do onRealtimeConnectRequest this hook  fires only for "example" collection
